build.gradle

```diff
@@ -10,6 +10,7 @@ allprojects {
     version gitVersion()
 
     apply plugin: "java"
+    apply plugin: "idea"
```

Comment:
how come you've introduced this here?

I think it would be a good thing if we started using `gradlew idea` to generate the IntelliJ dependencies instead of relying on the IntelliJ plugin which has caused people a lot of grief.

Reply:
Without this - IntelliJ can't find the protoc generated class files so won't autocomplete on them etc.

This plugin does seem to work really well!